test/parallel/test-webstorage.js

```diff
@@ -145,3 +114,14 @@ describe('webstorage quota for localStorage and sessionStorage', () => {
     assert.match(cp.stderr, /QuotaExceededError/);
   });
 });
+
+test('disabled with --no-experimental-webstorage', async () => {
+  for (const api of ['Storage', 'localStorage', 'sessionStorage']) {
+    const cp = await spawnPromisified(process.execPath, ['--no-experimental-webstorage', '-e', api]);
```

Comment:
```suggestion
    const cp = await spawnPromisified(process.execPath, ['--no-experimental-webstorage', '--localstorage-file', '<PATHHERE>',  '-e', api]);
```

To ensure it's noop as stated in the docs, it needs the `--localstorage-file` as well.

This PR looks good to me with the change.